docs/misp2_installation_manual_18.04.md

```diff
@@ -340,6 +342,77 @@ described in Section
 [5.3](#53-configuring-https-connection-between-misp2-application-and-x-road-security-server)
 of this guide.
 
+##### 4.3.2.1 Configuring support for the Estonian ID-card
```

Comment:
I'd consider moving the whole section from 4.3.2.1 to 5.7 since it's more about configuration rather than installation. Also, configuration of Mobile-ID is in section 5 too. I'd add it as a new subsection to avoid changing the existing numbering and breaking any external links.

In case the section is moved, I'd add something like this at the end of the section 4.3.2:

If the Estonian ID-card based authentication needs to be supported, please read the Section [5.7](#57-configuring-support-for-the-estonian-id-card) of this guide.

Reply:
Moved and added a notice to 4.3.2.